modules/Bio/EnsEMBL/Variation/TranscriptVariationAllele.pm

```diff
@@ -1594,6 +1595,16 @@ sub hgvs_protein {
   my $vf = $tv->base_variation_feature;
   my $tr          = $tv->transcript;
 
+  my $adaptor_shifting_flag = 1;
+
+  ## Check previous shift_hgvs_variants_3prime flag and act accordingly
+  $adaptor_shifting_flag = $vf->adaptor->db->shift_hgvs_variants_3prime() if (defined($vf->adaptor) && defined($vf->adaptor->db));
+  
+  my $hash_already_defined = defined($self->{shift_hash});
+  ## Perform HGVS shift even if no_shift is on
```

Comment:
Can you expand the comment to include the adaptor shift option, for e.g. `Perform HGVS shit if adaptor behaviour set for shifting even if transcript no_shift is on`